src/components/Data/Collections/CreateOrUpdate.vue

```diff
@@ -150,11 +154,24 @@
 </style>
 
 <script>
+import { validationMixin } from 'vuelidate'
+import { requiredUnless } from 'vuelidate/lib/validators'
+
 import Headline from '../../Materialize/Headline'
 import Focus from '../../../directives/focus.directive'
 import JsonEditor from '../../Common/JsonEditor'
 
+function isValidCollectionName(value) {
```

Comment:
same here, maybe we can put this function in the `methods` object

Reply:
same as above